apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -424,6 +453,14 @@ export default class TutorialExplorer extends React.Component {
           return false;
         }
 
+        if (
+          searchTerm !== '' &&
+          (!tutorial.name.toLowerCase().includes(searchTerm) ||
+            !tutorial.longdescription.toLowerCase().includes(searchTerm))
```

Comment:
Ooh, if tutorial.name and tutorial.longdescription are localized, this might "just work" for other languages!